dlc-oracle-test/src/test/scala/org/bitcoins/dlc/oracle/DLCOracleTest.scala

```diff
@@ -98,10 +99,13 @@ class DLCOracleTest extends DLCOracleFixture {
         assert(rValDbOpt.isDefined)
         val rValDb = rValDbOpt.get
         val hash = CryptoUtil.taggedSha256(
-          rValDb.nonce.bytes ++ CryptoUtil.serializeForHash(rValDb.eventName),
-          "DLCv0/Commitment")
+          rValDb.nonce.bytes ++ CryptoUtil.serializeForHash(
+            rValDb.eventName) ++ ByteVector.fromLong(
+            testEventDb.maturationTime.getEpochSecond),
```

Comment:
I'm not sure I'm happy with using a non-string serialization mixed with a string one, you should string append the long rather than the number's bytes

Reply:
Discussed on slack - Conclusion: fine for now, fix when we have a clear spec for the announcement signature